cmd/generate/main.go

```diff
@@ -183,7 +183,8 @@ func NewGenerateCommand() *cobra.Command {
 	cmd.Flags().StringVar(&outPath, "output", "/dev/stdout", "File to store the manifests")
 	cmd.Flags().StringVar(&params.StorageParams.AzureStorage.Container, "storage.azure.container", "azure", "Azure container(taken from storage secret)")
 	cmd.Flags().StringVar(&params.StorageParams.GCS.Bucket, "storage.gcs.bucket", "tempo", "GCS storage bucket (taken from storage secret)")
-	cmd.Flags().StringVar(&params.StorageParams.S3.Endpoint, "storage.s3.endpoint", "http://minio.minio.svc:9000", "S3 storage endpoint (taken from storage secret)")
+	cmd.Flags().StringVar(&params.StorageParams.S3.Endpoint, "storage.s3.endpoint", "minio.minio.svc:9000", "S3 storage endpoint (taken from storage secret)")
```

Comment:
Is this a breaking change?

Reply:
You're right, I updated the PR now to make it non-breaking.